doc/reference/configuration/cfg_replication.rst

```diff
@@ -482,6 +482,23 @@
     | Environment variable: TT_REPLICATION_SYNCHRO_TIMEOUT
     | Dynamic: **yes**
 
+.. confval:: replication_threads
+
+    Since version :doc:`2.10.0 </release/2.10.0>`.
+    The number of threads spawned to decode the incoming replication data.
+
+    The default value is `1`.
+    It means that a single separate thread handles all the incoming replication streams.
```

Comment:
Let's mention that a single thread for all the incoming data should be enough in almost all cases, so the user probably won't have to touch this config option.